### DISCUSSION TITLE
FIgures & confidence intervals

### DISCUSSION THREAD

**Discussion by vkhodygo** [MRC-CSO-SPHSU/covid19_effect_estimates#5] [Ideas]
@dkopasker @andrewbaxter439 @rachelmthomson 

I suggest we discuss everything related to the topic here. I've sent you the data for this already.

**Comment by andrewbaxter439** [comment]
Thanks Vlad for setting this up. Is it worth:
- syncing R version? I'm on 4.2.1 but can aim to keep consistent with what others are using
- syncing versions of dependants using `renv`? can set this up in review branch

**Comment by vkhodygo** [comment]
@andrewbaxter439 
> syncing R version? I'm on 4.2.1 but can aim to keep consistent with what others are using

I'm using it as well, it's also the version installed on the workstation if I remember correctly.

> syncing versions of dependants using renv? can set this up in review branch

That would be great, I usually stick to the latest versions, but that's not how it's supposed to be.

**Comment by andrewbaxter439** [comment]
Have opened an issue at #6 but can move back here if most helpful.

**Comment by vkhodygo** [comment]
@andrewbaxter439 
No, this place is for more general talks, having separate issues is not a bad idea.

General notes: 
- I think I've told you that already, but you should keep the changes in a separate branch, pushing directly to `master` is a bad habit even if you're the only one working on the project. I'm not going to revert the latest changes, it's counterproductive, but for the sake of the end user it's better to use multiple branches.
- No spaces in filenames, ever. Underscores are your best friends.

**Reply by andrewbaxter439** [reply]
kicking myself that I created a branch but then cloned/pushed to wrong one! have rebased changes to correct branch and pushed. feel free to revert `master` as I'll continue to work on `ab_rev` branch until merge-ready.

Will also change filename :+1:

**Reply by vkhodygo** [reply]
No worries, that happens sometimes.
